File: chapter2/c/ftoc.c
#include <stdio.h>

double ftoc(double value);

int main(void) {

	const double boiling = 212.0;
	const double freezing = 32.0;

	printf("%gF = %gC\n", boiling, ftoc(boiling)); 
	printf("%gF = %gC\n", freezing, ftoc(freezing)); 

	return 0;
}

double ftoc(double value) {
	return (value - 32) * 5 / 9;	
}

File: chapter2/go/ftoc.go
package main

import "fmt"

func main() {
	const freezing, boiling = 32.0, 212.0

	fmt.Printf("%gF = %gC\n", freezing, ftoc(freezing))
	fmt.Printf("%gF = %gC\n", boiling, ftoc(boiling))
}

func ftoc(f float64) float64 {
	return (f - 32) * 5 / 9
}
